curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables.md

```diff
@@ -95,15 +95,12 @@ if(typeof player !== "undefined"){(function(v){return v;})(player);}
 ## --seed-contents--
 
 ```js
-// Setup
 var testObj = {
   12: "Namath",
   16: "Montana",
   19: "Unitas"
 };
 
-// Only change code below this line
-
 var playerNumber;       // Change this line
 var player = testObj;   // Change this line
```

Comment:
```suggestion
var playerNumber;
var player = testObj;
```
I think I agree with Tom that this could be removed. IMHO the instructions are clear enough.